inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/settings/WorkflowAndMonitoringPanel.java

```diff
@@ -103,7 +105,8 @@ public void onSubmit(AjaxRequestTarget target)
                     workflowProperties.setActive(true);
                 }
 
-                setResponsePage(getApplication().getHomePage());
+                info("Workflow and workload settings changed");
```

Comment:
Feedback messages are only shown if the feedback panel is part of the request:

`aTarget.addChildren(getPage(), IFeedback.class);`